pkg/agent/types/pod_encap_mode.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"net"
+	"strings"
+)
+
+type PodEncapMode int
```

Comment:
I feel TrafficEncapMode is better. It can be Node -> Pod traffic, and even Node -> GW traffic as well.

Reply:
done